src/govuk/components/radios/radios.js

```diff
@@ -37,21 +36,35 @@ Radios.prototype.init = function () {
 }
 
 Radios.prototype.setAttributes = function ($input) {
-  var inputIsChecked = $input.checked
-  $input.setAttribute('aria-expanded', inputIsChecked)
+  var $content = document.querySelector('#' + $input.getAttribute('aria-controls'))
```

Comment:
Non-blocking: It could be worth leaving a comment here to say that we use `document` as the scope to cater for the edge case when other radios with conditionals are part of the same group. It might not be immediately clear that there could be other radios that are in fact related even though they're outside `$module`.

Reply:
I originally put a comment in but @36degrees thought that if we were doing this from scratch that we probably wouldnt have commented it. The case you're talking about is described in the test suite, so maybe that's enough? What do you think?

Reply:
I think if someone changed this the tests would fail so I'm going to go with what Ollie and I discussed originally so can get this merged 🐎